tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2018",
```

Comment:
If I remember correctly, this is setting the compile output target for JS from TS transpiler.

Was this intentional? Jumping from ES5 (2009) to ES2018 seems like a big compatibility ask.

Reply:
This is only on the server code, as we already depend on node >= 8, we have no issues here, the advantage is that we can now iterate over ES6 `Map`, before we couldn't ;)

Reply:
Sweeeeeet!